src/framed/std/core.cljc

```diff
@@ -130,3 +139,15 @@
        (loop []
          ~@body
          (recur)))))
+
+(defn try-to-edn
```

Comment:
The naming was very tricky; I don't like the `try` implication either, I considered `maybe` or `guard` but felt this came out better